py4j-java/src/main/java/py4j/CallbackConnection.java

```diff
@@ -112,11 +118,12 @@ public CallbackConnection(int port, InetAddress address, SocketFactory socketFac
 	 * @param authToken Token for authenticating with the callback server.
 	 */
 	public CallbackConnection(int port, InetAddress address, SocketFactory socketFactory, int readTimeout,
-			String authToken) {
+			int createSocketTimeout, String authToken) {
```

Comment:
```suggestion
			int connectTimeout, String authToken) {
```

also avoid API break by keeping old constructor signature with 🔭 constructor